docs/docs/deploy/introduction.mdx

```diff
@@ -19,7 +19,7 @@ Are you unfamiliar with Docker, Kubernetes and Helm? Check out "[Understanding R
 ## When to Deploy Your Assistant
```

Comment:
Weird syntax highlighting in GitHub UI. Maybe your editor introduced weird control characters?! and maybe that's related to the issue you mentioned [here](https://rasa-hq.slack.com/archives/C047U7ZBKM1/p1695199926901089?thread_ts=1695131778.282159&cid=C047U7ZBKM1). Just suppositions
<img width="956" alt="Screenshot 2023-09-20 at 14 20 21" src="https://github.com/RasaHQ/rasa/assets/1931615/a58cb480-7e87-45d1-82a0-dde2444029fc">

Reply:
thanks for the suggestion. doesn't look like any control chars in here though.